policies/templates/gcp_gke_cluster_version_v1.yaml

```
@@ -0,0 +1,142 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-gke-cluster-version-v1
+spec:
+  crd:
+    spec:
+      names:
+        kind: GKEClusterVersionConstraintV1
+      validation:
+        openAPIV3Schema:
+          properties:
+            mode:
+              type: string
+              enum: [denylist, allowlist]
+              description: "String identifying the operational mode, allowlist or denylist. In allowlist mode,
+              GKE clusters are only allowed to have a master version specified in the 'master_versions' parameter. In denylist mode,
+              resources are allowed any master versions except those listed in the 'master_versions' parameter.
+              Default is allowlist."
+            version_type:
+              type: string
+              enum: [master, node]
+              description: "String identifying the GKE cluster version type to look for, master or node.
+              master refers to the Kubernetes version for GKE cluster masters (currentMasterVersion).
+              node refers to the Kubernetes version for GKE cluster nodes (currentNodeVersion).
+              Default is master."
+            exemptions:
+              type: array
+              items: string
+              description: "Array of GKE clusters in the form of glob expressions to exempt from version restriction.
+              Delimiter is '/'.
+              E.g. `//container.googleapis.com/projects/project-abc/zones/*/clusters/*` will exempt any clusters
+              whose asset name has the format '//container.googleapis.com/projects/project-abc/zones/{ANY ZONE}/clusters/{ANY NAME}'.
+              such as:
+              '//container.googleapis.com/projects/project-abc/zones/us-central1-c/clusters/my-cluster'
+              but NOT
+              '//container.googleapis.com/projects/wrong-project/zones/us-central1-c/clusters/wrong-cluster'"
+            versions:
+              type: array
+              items: string
+              description: "Array of versions that GKE clusters are allowed to have,
+              e.g. 1.12.10-gke.17 for currentMasterVersion or 1.4.4 for currentNodeVersion."
```

Shouldn't refer to "currentMasterVersion"

```suggestion
              e.g. 1.12.10-gke.17 or 1.4.4
```

---

Ack